### ISSUE TITLE
multiple routes on a single url ?

### ISSUE THREAD

**Issue by seriousme** [fission/fission#102]
I'm not sure this is intended behaviour:

If I add two functions to the same URL they are both accepted by fission and fission will execute the first one, however when listing functions the order is reversed and its not clear which one is active
```
fission route create --method GET --url /hello --function hello
trigger 'd09e4d12-5e5b-4ebd-bf66-c4cd67e846ae' created
fission route create --method GET --url /hello --function hello2
trigger '59add1e2-630d-42dc-ab7e-e993a7de4e77' created
fission route list
NAME                                 METHOD URL    FUNCTION_NAME FUNCTION_UID
59add1e2-630d-42dc-ab7e-e993a7de4e77 GET    /hello hello2
d09e4d12-5e5b-4ebd-bf66-c4cd67e846ae GET    /hello hello
```

in this setup ```hello``` is executed and not ```hello2``` , only after removing the route to hello , hello2 takes over

**Comment by soamvasani** [comment]
Definitely a bug. The controller should do better validation for routes.

**Comment by soamvasani** [comment]
Fixed by #111
